drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/plugins/ConditionColumnPlugin.java

```diff
@@ -199,7 +199,7 @@ void prepareValues() {
             editingCol().setOperator(null);
         }
 
-        if (constraintValue() != BaseSingleFieldConstraint.TYPE_LITERAL) {
+        if (!(tableFormat() == GuidedDecisionTable52.TableFormat.LIMITED_ENTRY || constraintValue() == BaseSingleFieldConstraint.TYPE_LITERAL)) {
```

Comment:
Hi @karreiro , just question, this condition is quite complex and it seems to me that it should be kept in sync with `valueOptionsPage.canSetupBinding()`. I see that `canSetupBinding()` and this `if` are sligtly different, however, what do you think , could this `if` looks like `if (`valueOptionsPage.canSetupBinding())`?